src/typical.jl

```diff
@@ -90,7 +90,14 @@ function _replace(term::InteractionTerm, typicals::Dict)
 end
 
 _termsyms(t) = StatsModels.termsyms(t)
-_termsyms(t::FunctionTerm{Fo,Fa,Names}) where {Fo,Fa,Names} = Names
+
+@static if hasfield(FunctionTerm, :args)
+    # StatsModels 0.7
+    _termsyms(t::FunctionTerm) = StatsModels.termsyms.(only(t.args).args)
```

Comment:
what's happening with the `only` here?  I'd have thought you want 
```suggestion
    _termsyms(t::FunctionTerm) = StatsModels.termsyms.(t.args)
```

Reply:
I was trying to test locally and might have had a problematic example.

Reply:
oh might need a `reduce(union)` around this...